packages/components/card/src/utils/constants.js

```diff
@@ -0,0 +1,22 @@
+export const orientations = {
+	horizontal: 'horizontal',
```

Comment:
Most of the time we set the key of constants as uppercase. For example `HORIZONTAL:horizontal`. If there are two words, we use `_` to connect them.

Reply:
Okay, I'll change them 👍